opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -354,7 +367,10 @@ impl<R: RuntimeChannel> BatchSpanProcessorInternal<R> {
                     let export_task = self.export();
                     let task = async move {
                         if let Err(err) = export_task.await {
-                            global::handle_error(err);
+                            otel_error!(
+                                name: "BatchSpanProcessor.Export.Error",
```

Comment:
There is another log with the name "BatchProcessor.OnEnd.Error". Is this duplicate ?

Reply:
Good catch. We have `SimpleProcessor.OnEnd.Error`. To make it consistent, this can be modified to `BatchProcessor.OnEnd.Error`

Reply:
These are the event types:

`SimpleProcessor.OnEnd.Error`: export error
`BatchSpanProcessor.OnEnd.Error`: export error
`BatchSpanProcessor.OnEnd.SendResultError`: error sending export status/